projects/1.21/assets/expanded-delight/expandeddelight/lang/zh_cn.json

```
@@ -0,0 +1,144 @@
+{
+  "item.expandeddelight.asparagus": "芦笋",
+  "item.expandeddelight.asparagus_seeds": "芦笋种子",
+  "item.expandeddelight.asparagus_soup": "芦笋汤",
+  "item.expandeddelight.asparagus_soup_creamy": "奶油芦笋汤",
+  "item.expandeddelight.asparagus_and_bacon_cheesy": "芝士芦笋培根",
+  "item.expandeddelight.asparagus_mushroom_pasta": "芦笋蘑菇意面",
+
+  "item.expandeddelight.sweet_potato": "红薯",
+  "item.expandeddelight.baked_sweet_potato": "烤红薯",
+  "item.expandeddelight.sweet_potato_salad": "红薯沙拉",
+  "item.expandeddelight.sweet_potato_casserole": "红薯砂锅",
+
+  "item.expandeddelight.chili_pepper": "辣椒",
+  "item.expandeddelight.chili_pepper_seeds": "辣椒种子",
+  "item.expandeddelight.peperonata": "炖辣椒",
+
+  "item.expandeddelight.peanut": "花生",
+  "item.expandeddelight.peanut_butter": "花生酱",
+  "item.expandeddelight.peanut_butter_sandwich": "花生酱三明治",
+  "item.expandeddelight.peanut_butter_honey_sandwich": "花生酱蜂蜜三明治",
+  "item.expandeddelight.peanut_salad": "花生沙拉",
+  "item.expandeddelight.peanut_honey_soup": "蜂蜜花生汤",
+  "item.expandeddelight.sweet_berry_jelly_sandwich": "花生酱甜浆果果冻三明治",
+  "item.expandeddelight.glow_berry_jelly_sandwich": "花生酱荧光浆果果冻三明治",

```

```suggestion
  "item.expandeddelight.sweet_berry_jelly_sandwich": "花生酱甜浆果果酱三明治",
  "item.expandeddelight.glow_berry_jelly_sandwich": "花生酱发光浆果果酱三明治",
```
jelly在这里应该指果酱，请通查
“发光浆果”为原版用词